### PR TITLE
Fix wrong option in cmatrix.rb

### PR DESCRIPTION
**Pull request by jam7**
I cannot compile cmatrix with original package file.  I needed to change the way to set environment variable like this PR.

Tested on armv7l and x86_64.

### CODE REVIEW

File: packages/cmatrix.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Cmatrix < Package
   depends_on 'ncurses'
 
   def self.build
-    system './configure --prefix=/usr/local CPPFLAGS="-I/usr/local/include/ncurses"'
+    system 'CPPFLAGS=-I/usr/local/include/ncurses ./configure --prefix=/usr/local'
```
Comment (uberhacker) [inline packages/cmatrix.rb]: `system "CPPFLAGS=-I#{CREW_PREFIX}/include/ncurses ./configure --prefix=#{CREW_PREFIX}"` ?

Comment (jam7) [inline packages/cmatrix.rb]: done

**Comment by uberhacker** [comment]
Bump version to detect changes?

**Comment by jam7** [comment]
done

**Pull request merged**